unitTests/ui/DynamicFacet/DynamicFacetRangeTest.ts

```diff
@@ -80,13 +80,20 @@ export function DynamicFacetRangeTest() {
       expect(test.cmp.facetType).toBe(FacetType.numericalRange);
     });
 
-    it(`when not setting a valueFormat option
-      should set it number`, () => {
+    it(`when not setting a valueFormat option with a field containing "date"
```

Comment:
Should the "not" be removed from this test description?

Reply:
Sorry I'm a recovering copy and paster

Reply:
valueFormat is not set by default